Sources/Defaults/Defaults.swift

```diff
@@ -1,12 +1,12 @@
 // MIT License © Sindre Sorhus
 import Foundation
 
-public protocol _DefaultsBaseKey: Defaults.Keys {
+public protocol DefaultsBaseKey: Defaults.Keys {
```

Comment:
Can you add `typealias BaseKey = DefaultsBaseKey` to line 17 and then use `BaseKey` throughout?